### PR TITLE
fix(controller): enhanced connection process

### PR DESCRIPTION
**Pull request by Tbaile**
Modal: 

https://github.com/user-attachments/assets/280ec031-cf49-4185-8751-917a9e9c3b3a

Ref:
- https://github.com/NethServer/nethsecurity/issues/754

### CODE REVIEW

File: public/i18n/en/translation.json [hunk 1]
```diff
@@ -777,7 +777,15 @@
       "vpn_ip_address": "VPN IP address",
       "disconnect": "Disconnect",
       "disconnect_unit_message": "The unit will be disconnected from the controller and will no longer be accessible from the controller itself. Do you want to proceed?",
-      "data_updated_every_seconds": "Data updated every {seconds} seconds"
+      "data_updated_every_seconds": "Data updated every {seconds} seconds",
+      "disclaimer": "Disclaimer",
+      "connect_unit_message": "Unit logs will be transmitted to the Controller for the purposes of storage, monitoring, and analysis of network usage. \nIf you do not wish for this data to leave the device, do not connect to the Controller.",
+      "confirm_and_connect": "Confirm and connect",
+      "sending_data_to_controller": "Sending Monitoring data to Controller",
```
Comment (andre8244) [inline public/i18n/en/translation.json:784]:
```suggestion
      "sending_data_to_controller": "Sending monitoring data to Controller",
```

File: public/i18n/en/translation.json [hunk 2]
```diff
@@ -777,7 +777,15 @@
...
+      "sending_data_to_controller_tooltip": "Complete monitoring available on Grafana with NethSecurity",
```
Comment (andre8244) [inline public/i18n/en/translation.json:785]: Let's use [linked messages](https://vue-i18n.intlify.dev/guide/essentials/syntax#linked-messages) to build the string with the link to subscription: different languages might use a different position for the word "subscription".

File: public/i18n/en/translation.json [hunk 3]
```diff
@@ -777,7 +777,15 @@
...
+      "sending_data_to_controller_tooltip_link": "subscription",
+      "last_sent": "Last sent: {date}.",
```
Comment (andre8244) [inline public/i18n/en/translation.json:787]:
```suggestion
      "last_sent": "Last sent: {date}",
```

File: public/i18n/en/translation.json [hunk 4]
```diff
@@ -777,7 +777,15 @@
...
+      "no_last_sent": "Data is being prepared for transmission."
```
Comment (andre8244) [inline public/i18n/en/translation.json:788]:
```suggestion
      "no_last_sent": "Data is being prepared for transmission"
```

File: public/i18n/en/translation.json (hunk 2 above)
Comment (Tbaile) [reply public/i18n/en/translation.json:785]: Fine by me, but how do we set the link for a translated variable in the tooltip?

Comment (andre8244) [reply public/i18n/en/translation.json:785]: Sorry I linked the wrong docs page, check this out: https://vue-i18n.intlify.dev/guide/advanced/component.html#slots-syntax-usage

You can also find other usage examples by searching `<i18n-t` inside the repo source

Comment (Tbaile) [reply public/i18n/en/translation.json:785]: Never used the component, very nice to have!

File: public/i18n/en/translation.json [hunk 5]
```diff
@@ -777,7 +777,15 @@
       "vpn_ip_address": "VPN IP address",
       "disconnect": "Disconnect",
       "disconnect_unit_message": "The unit will be disconnected from the controller and will no longer be accessible from the controller itself. Do you want to proceed?",
-      "data_updated_every_seconds": "Data updated every {seconds} seconds"
+      "data_updated_every_seconds": "Data updated every {seconds} seconds",
+      "disclaimer": "Disclaimer",
+      "connect_unit_message": "Unit logs will be transmitted to the Controller for the purposes of storage, monitoring, and analysis of network usage. \nIf you do not wish for this data to leave the device, do not connect to the Controller.",
```
Comment (andre8244) [inline public/i18n/en/translation.json:782]: I'd split this in two messages instead of using `\n`, to prevent translation errors

**Review by andre8244 — APPROVED** [review]
Just an optional 18n enhancement

File: public/i18n/en/translation.json (hunk 5 above)
Comment (Tbaile) [reply public/i18n/en/translation.json:782]: The char was not intended to be there in the first place :sweat_smile: 
Probably a copy-paste issue

**Pull request merged**